platforms/core-execution/build-cache-http/src/main/java/org/gradle/caching/http/HttpBuildCache.java

```diff
@@ -129,19 +106,16 @@ public void credentials(Action<? super HttpBuildCacheCredentials> configuration)
      *
      * @since 4.2
      */
-    @ToBeReplacedByLazyProperty
-    public boolean isAllowUntrustedServer() {
-        return allowUntrustedServer;
-    }
-
-    /**
-     * Specifies whether it is acceptable to communicate with an HTTP build cache backend with an untrusted SSL certificate.
-     *
-     * @see #isAllowUntrustedServer()
-     * @since 4.2
-     */
-    public void setAllowUntrustedServer(boolean allowUntrustedServer) {
-        this.allowUntrustedServer = allowUntrustedServer;
+    @ReplacesEagerProperty(originalType = boolean.class)
+    public abstract Property<Boolean> getAllowUntrustedServer();
+
+    // kotlin source compatibility
```

Comment:
💅 Can we use Javadoc instead of normal comments, why just // instead of `/****/`? Also probably TODO is not needed here